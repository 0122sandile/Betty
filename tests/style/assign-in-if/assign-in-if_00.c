int main(void)
{
	int i;

	if ((i = 0) == 1)
	{
		return (1);
	}
	return (0);
}
