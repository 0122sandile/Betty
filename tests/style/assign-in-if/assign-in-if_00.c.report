Mismatch output for assign-in-if_00.c

Output:
assign-in-if_00.c: line 5: do not use assignment in if condition [--assign-in-if]

Expected:
assign-in-if_00.c: lin 5: do not use assignment in if condition [--assign-in-if]
