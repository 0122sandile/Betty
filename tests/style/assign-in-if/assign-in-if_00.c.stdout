assign-in-if_00.c: lin 5: do not use assignment in if condition [--assign-in-if]
